oneview_redfish_toolkit/app.py

```diff
@@ -62,3 +62,42 @@
 app.register_blueprint(manager)
 app.register_blueprint(odata)
 app.register_blueprint(thermal)
+
+if __name__ == "__main__":
+
+    config = util.config
+
+    try:
+        port = int(config["redfish"]["redfish_port"])
+    except Exception:
+        logging.error("Port must be an integer number between 1 and 65536")
+        exit(1)
+    # Checking port range
+    if port < 1 or port > 65536:
+        logging.error("Port must be an integer number between 1 and 65536")
+        exit(1)
+
+    ssl_type = config["ssl"]["SSLType"]
+    # Check SSLType:
+    if ssl_type not in ('disabled', 'adhoc', 'certs'):
+        logging.error(
+            "Invalid SSL type: {}. Must be one of: None, adhoc or certs".
```

Comment:
Instead of None here it should say disabled.

Reply:
fixed